src/validations/test/rules/sar.xspec

```diff
@@ -296,6 +296,206 @@
 
         </x:scenario>
 
+        <x:scenario
+            label="when the metadata">
+            <x:scenario
+                label="has a responsible-party of role-id 'penetration-test-lead">
+                <x:scenario
+                    label="and there is one party-uuid child">
+                    <x:context>
+                        <assessment-results
+                            xmlns="http://csrc.nist.gov/ns/oscal/1.0">
+                            <metadata>
+                                <responsible-party
+                                    role-id="penetration-test-lead">
+                                    <party-uuid>e934d8b5-13e5-4f77-b55e-871e6f2df2fe</party-uuid>
+                                </responsible-party>
+                            </metadata>
+                        </assessment-results>
+                    </x:context>
+                    <x:expect-not-assert
+                        id="has-pen-test-lead"
+                        label="this is correct" />
+                </x:scenario>
+                <x:scenario
+                    label="and there is zero or more than one party-uuid child">
+                    <x:context>
+                        <assessment-results
+                            xmlns="http://csrc.nist.gov/ns/oscal/1.0">
+                            <metadata>
+                                <responsible-party
+                                    role-id="penetration-test-lead">
+                                    <party-uuid>e934d8b5-13e5-4f77-b55e-871e6f2df2fe</party-uuid>
+                                    <party-uuid>e934d8b5-13e5-4f77-b55e-871e6f2df2ff</party-uuid>
+                                </responsible-party>
+                            </metadata>
+                        </assessment-results>
+                    </x:context>
+                    <x:expect-assert
+                        id="has-pen-test-lead"
+                        label="this is incorrect" />
```

Comment:
this is in a few places, but I think most of the asserts use this language. It reads fine either was, so 🤷 

```suggestion
                        label="that is incorrect" />
```

Reply:
I feel like, if this/that is desired we should just do a global SR on the xpsecs and commit that as a single story.  I'll start an issue.